3rdparty/lua-5.1.3/src/loslib.c

```diff
@@ -1,15 +1,16 @@
 /*
-** $Id: loslib.c,v 1.19.1.3 2008/01/18 16:38:18 roberto Exp $
-** Standard Operating System library
-** See Copyright Notice in lua.h
-*/
+ ** $Id: loslib.c,v 1.19.1.3 2008/01/18 16:38:18 roberto Exp $
```

Comment:
sorry about this, it's a copy/paste auto-indent thing. i'll revert and add only the actual necessary change.